content/en/contribute/docs/style-guide.md

```diff
@@ -144,13 +144,18 @@ Connecting readers to related content in different pages is an important aspect
 1. **Inline links**: a portion of any narrative text can link to another page. This should done using the markdown link notation. 
    
    For example, the text `linking documents is a [foundational reason for the web existing in the first place](https://en.wikipedia.org/wiki/Hypertext)!` yields: "linking documents is a [foundational reason for the web existing in the first place](https://en.wikipedia.org/wiki/Hypertext)!"
-2. **See Also**: the `see-also` shortcode is available to connect to an important concept within the documentation site. The link will be more prominent to the reader by having a common prefix and shown on a separate line.
+   
+1. **See Also**: the `see-also` shortcode is available to connect to an important concept within the documentation site. The link will be more prominent to the reader by having a common prefix and shown on a separate line.
    
    For example, `{{</* see-also page="design/icons" */>}}` will show as seen here: {{< see-also page="design/icons" >}}
+   
+   You can also make the callout say "Read More" with the `prefix` tag: `{{</* see-also prefix="Read More" page="design/icons" */>}}`. This will show as seen here: {{< see-also page="design/icons" prefix="Read More" >}} 
+   
+   A custom title and anchor can be provided with this as well. For example, `{{</* see-also page="design/icons" title="Learn about the Icon Library" anchor="about-the-icon-library" */>}}`, will show as: {{< see-also page="design/icons" title="Learn about the Icon Library" anchor="about-the-icon-library" >}}
+   
+   Please use `see-also` when reference __related topics__ (e.g. [Workflows]({{< ref "apps/concepts/workflows" >}})) and use `read-more` when referencing the __same topic__ in more depth  (e.g. [Home Page]({{< ref "/" >}})) .  
```

Comment:
We don't use `__text__` for bold (much?) anywhere else in documentation, and don't have it listed as such in the [docs style guide](https://github.com/medic/cht-docs/blob/74d07696a3c41de337b1673e850663862e5ef32a/content/en/contribute/docs/style-guide.md#use-markdown-notation). It is a bit arbitrary, but unless there is a clear reason to use double underscores it would be nice to stay consistent with the styling of the page and use `**text**` here. 

We can also add this as a do or don't in the style guide if you think that would help!

Reply:
Oop! I'd actually meant them to be italics, not bold. Moved back to italics, but am happy to go bold (via `**`) if that's preferred!